.github/scripts/codespell.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -ex
+
+SCAN_RESULT=`codespell -H --quiet-level=3 --skip="*.h,*.cmake,*.c,*.hpp,*.cpp,*.sh,*.py,test/supp/*.supp,./.venv" --ignore-words-list="ASSER,Tne,ba,BA"`
```

Comment:
Are there plenty of false-positives in source code files, like i.e. .h, .c, .cpp? This spell checker would be very useful in these files, too.

Reply:
Done